include/seeds.scheduler.hpp

```diff
@@ -89,11 +103,17 @@ CONTRACT scheduler : public contract {
         DEFINE_CONFIG_TABLE_MULTI_INDEX
 
         typedef eosio::multi_index < "operations"_n, operations_table,
-            indexed_by<"bytimestamp"_n, const_mem_fun<operations_table, uint64_t, &operations_table::by_timestamp>>
+            indexed_by<"bytimestamp"_n, 
+            const_mem_fun<operations_table, uint64_t, &operations_table::by_timestamp>>
```

Comment:
we probably need to do a migration for this table upgrade

We would not need to if we can just reset everything - and maybe that's the best way. 

But there were some ops we don't reset so that the cycles remain intact. 

i.e. example the cycle is 1 week, it fired 3 days ago, 4 days to go

If we now reset this op, it will fire in 7 days instead of 4 days

Reply:
But, since the table data structure doesn't change - maybe it's good enough to take everything out, and put everything back in, in 1 action.

Reply:
Oh sorry, I didn't add a new index for the operations table, I just pressed \n so all the tables were indented in the same way.

Reply:
Oh, ok!